### DISCUSSION TITLE
REST api to sync a fork

### DISCUSSION THREAD

**Discussion by Milo123459** [community/community#29540] [Apps, API and Webhooks] [answered]
Is there a rest api to sync a fork? What permissions does it require on a PAT?

**Comment by mazkdevf** [comment, marked as answer]
Hello Milo,

Are you looking this: https://docs.github.com/en/rest/branches/branches#sync-a-fork-branch-with-the-upstream-repository ?

**Reply by Milo123459** [reply]
Yup! Thank you!

**Reply by Milo123459** [reply]
Is there an API to check if a fork is out-of-sync?

**Reply by mazkdevf** [reply]
> Is there an API to check if a fork is out-of-sync?

Uhm Lemme check!

**Reply by mazkdevf** [reply]
> Is there an API to check if a fork is out-of-sync?

Sorry i didn't find any, I could still be wrong. But hopefully someone who is little bit better at these things can tell you if its existing.

But for now Have a Good Day!

**Reply by Milo123459** [reply]
No problem, thank you so much for the help!

**Reply by mazkdevf** [reply]
> No problem, thank you so much for the help!

np
